lib/vmpooler/pool_manager.rb

```diff
@@ -337,6 +337,10 @@ def check_pool(pool)
                 check_ready_vm(vm, pool['name'], pool['ready_ttl'] || 0)
               rescue
               end
+            else
+              $logger.log('d', "[*] [#{pool['name']}] vm #{vm} is not in inventory, removing from 'ready' queue")
```

Comment:
Can you modify this line to log the VM hostname in quotes?


Currently:
````
[2015-04-07 14:58:48] [ ] [debian-7-x86_64] 'strra6rm2hgbm93' is being cloned from 'debian-7-x86_64'
[2015-04-07 14:58:53] [+] [debian-7-x86_64] 'strra6rm2hgbm93' cloned from 'debian-7-x86_64' in 5.17 seconds
[2015-04-07 14:59:26] [*] [debian-7-x86_64] vm strra6rm2hgbm93 not in inventory, removing from 'ready' queue
````

Should be:

````
[2015-04-07 14:59:26] [*] [debian-7-x86_64] 'strra6rm2hgbm93' not found in inventory, removing from 'ready' queue
````

Otherwise I've tested and am :+1: on this.